packages/connect-examples/new-example/app/components/common/DeviceInteractionArea.tsx

```
@@ -0,0 +1,289 @@
+                        alt={`OneKey ${currentDevice.deviceType || 'Device'}`}
+                        className="max-w-full max-h-full object-contain filter drop-shadow-lg"
+                      />
+                    </div>
+                  ) : (
+                    /* 默认设备图标 */
+                    <div className="w-24 h-36 bg-gradient-to-b from-gray-50 to-gray-100 dark:from-gray-800 dark:to-gray-900 rounded-lg border border-gray-200 dark:border-gray-700 shadow-sm relative overflow-hidden">
+                      {/* 屏幕区域 */}
+                      <div className="absolute top-3 left-3 right-3 h-20 bg-gray-900 dark:bg-gray-100 rounded-sm"></div>
+
+                      {/* 按钮区域 */}
+                      <div className="absolute bottom-3 left-1/2 transform -translate-x-1/2 w-8 h-2 bg-gray-300 dark:bg-gray-600 rounded-full" />
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)